File: src/main/java/peerlink/peerlink/dto/ChatDto.java
package peerlink.peerlink.dto;

import lombok.Getter;
import lombok.Setter;
import lombok.ToString;
import peerlink.peerlink.db.model.Message;

@Getter
@Setter
@ToString
public class ChatDto {
  private Message message;
  private String jwtToken;
}

File: src/main/client/peerlink/src/component/feature/chat/ChatDto.ts
export class Message {
  sender: string;
  receiver: string;
  content: string;
  constructor(
    sender: string = "",
    receiver: string = "",
    content: string = "",
    jwtToken: string = ""
  ) {
    this.sender = sender;
    this.receiver = receiver;
    this.content = content;
  }
}

export class ChatDto {
  message: Message;
  jwtToken: string;
  constructor(message: Message, jwtToken: string = "") {
    this.message = message;
    this.jwtToken = jwtToken;
  }
}
